src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyFulfillmentOrder.java

```diff
@@ -0,0 +1,349 @@
+package com.shopify.model.fulfillmentOrderApi;
```

Comment:
These models are so verbose. Graqhql is so much easier. We should really start considering getting that in here.

Reply:
That'd be fun!
Setting up is kinda difficult, but way easier to maintain once is up...
You have my vote for this